recipes/qcustomplot/all/conanfile.py

```diff
@@ -39,17 +40,14 @@ def config_options(self):
     def configure(self):
         if self.options.shared:
             self.options.rm_safe("fPIC")
-        # FIXME: we shouldn't have to force shared in qt, but config file
-        #        generated by conan in qt static is likely broken, or maybe env vars.
-        self.options["qt"].shared = True
+        self.options["qt"].gui = True
+        self.options["qt"].widgets = True
```

Comment:
This will either generate missing binaries in the CI, or if it's a hard requirement, also be ignored if the user sets something else thru the `!important` options introduced in Conan 2. this should be checked in the validate method if so :)

Reply:
There's already a validation check for them in place and they are enabled by default anyway in the qt recipe.

Reply:
qcustomplot should not count on it, each package should independent from each other. In case it requires a specific option, the same should be validated as well. You need to consider users can customize their profiles and have different results than those from CCI. Please, update validate() with that validation.

Reply:
Perfect, thank you!